commit 01bdcd6b55a0848e14d7b75625730dc66ce6ba6e
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-07-11T13:47:31-07:00 (3 months ago)
    
    TRITON-1810 skip vmapi user migration test when user migrations are allowed globally

diff --git a/test/lib/migration.js b/test/lib/migration.js
index f91e363..cfc414c 100644
--- a/test/lib/migration.js
+++ b/test/lib/migration.js
@@ -239,6 +239,13 @@ function TestMigrationCfg(test, cfg) {
     };
 
     test.bad_user_migrate_not_allowed_by_vm_default = function (t) {
+        // Check if the vmapi config allows migrations by default, if is is
+        // enabled then we can skip this test.
+        if (common.config.user_migration_allowed === true) {
+            t.ok(true, 'Skip - user migrations are enabled globally');
+            t.done();
+            return;
+        }
         // Should not be able to migrate a zone with
         // user_migration_allowed unset
         client.post({
